pkg/test/testPackages.proj

```diff
@@ -4,7 +4,7 @@
 
   <ItemGroup>
     <TestPackages Condition="'$(TestPackages)' != ''" Include="$(TestPackages)" />
-    <ExcludePackages Include="CoreFx.Private.TestUtilities" />
+    <ExcludePackages Include="CoreFx.Private.TestUtilities;System.IO.Ports;runtime.native.System.IO.Ports" />
```

Comment:
Won't you still want package testing to happen on System.IO.Ports and just remove it from the runtime specific package?

Reply:
System.IO.Ports depends on the runtime.native.* and that depends on the packages which don't exist yet. I'll try to fix it all in the next PR, let me test if the package is correct first and once the official package is up we can remove some of the hacks from here